src/gluonts/sagemaker/defaults.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""Placeholder docstring"""
+from __future__ import absolute_import
+
+# Standard library imports
+import os
+from pathlib import Path
+
+GLUONTS_VERSION = "0.3.3"
```

Comment:
Maybe. But the idea is that if one doesnt specify one it defaults to this, so when rerunning code with a different gluonts version this could break old code?